tests/test_managedblockchain/test_managedblockchain_networks.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import unicode_literals
+
+import boto3
+import sure  # noqa
+
+from moto import mock_managedblockchain
+
+
+@mock_managedblockchain
+def test_create_network():
```

Comment:
Would you mind adding a few negative tests as well? I.e., test_invalid_framework, test_invalid_framework_version, etc.
Ideally also a test to verify what happens if `get_network` is called with an unknown NetworkId

Reply:
Updated